docs/genai/api/c.md

```diff
@@ -101,6 +101,18 @@ Encodes a single string and adds the encoded sequence of tokens to the OgaSequen
 OGA_EXPORT OgaResult* OGA_API_CALL OgaTokenizerEncode(const OgaTokenizer*, const char* str, OgaSequences* sequences);
 ```
 
+### ApplyChatTemplate
+
+Processes the specified template with the provided input using the tokenizer, and outputs the resulting string. Optionally, it can include a generation prompt in the output. out_string must be freed with OgaDestroyString
+
+#### Parameters
```

Comment:
no params or returns for anything in the c.md file (not sure if this is by design or just not done yet), so omitted this here as well for now.